add-chain/cmd/compress-genesis.go

```diff
@@ -0,0 +1,255 @@
+package cmd
+
+import (
+	"bytes"
+	"compress/flate"
+	"compress/gzip"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	"github.com/ethereum-optimism/superchain-registry/add-chain/flags"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/urfave/cli/v2"
+)
+
+var CompressGenesisCmd = cli.Command{
```

Comment:
Do you think we need a separate sub command for this, or could it just run as a part of the main command?

Reply:
This could run as part of the main command, it was more of a personal preference to use a sub command. However, for the current e2e tests we have several rollup.json examples and only one genesis.json. Its nice for testing to avoid having multiple of the ~8MB genesis.json files stored in the repo. Maybe if we add `compress-genesis` command to the main command we add a flag for `--skip-compress-genesis=true` for flexibility?

I also think its nice to not overload the main command with more responsibility. But I'm open to putting them together if you feel like it leads to a better user experience.

Reply:
I don't feel strongly, sounds like there are some good reasons for keeping it as a subcommand. If we needed to add a flag when combining the commands, we aren't really simplifying overall.